packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx

```diff
@@ -343,12 +385,23 @@ export class CommandBar extends BaseComponent<ICommandBarProps, ICommandBarState
       let item = renderedItems[i];
       let itemWidth = this._commandItemWidths![item.key];
 
+      if (!itemWidth) {
+        continue;
+      }
+
       if ((consumedWidth + itemWidth) >= availableWidth) {
         if (i > 0 && !isOverflowVisible && (availableWidth - consumedWidth) < OVERFLOW_WIDTH) {
           i--;
         }
 
-        renderedOverflowItems = renderedItems.splice(i).concat(overflowItems!);
+        // Insert divider between overflow items
+        renderedOverflowItems = renderedItems
```

Comment:
I'm not sure that it should always add a divider between the rendered items and the overflow. Someone from design might know better. Right now most command bars we have don't have a divider.